src/runMocha.js

```diff
@@ -14,6 +15,11 @@ const runMocha = ({ config, testPath, globalConfig }, workerCallback) => {
       const failures = [];
       const passes = [];
 
+      runner.on('suite', () => {
+        if (clearMocks) {
```

Comment:
maybe only add the listener if `clearMocks` is set?

Reply:
When creating reporter, we don't know if clearMocks exists yet.

We can move creating a reporter to after requiring setupFilesAfterEnv - but that's not like it is implemented in other runners, and I'm not sure if it will be correct.